src/tools/src/settings.h

```diff
@@ -23,8 +23,9 @@ struct settings {
   bool            level_set         = false;
   int             global_level      = 1;
   bool            list_all          = false;
-  int     color          = 1;
-  double  alpha          = 1;
+  int             color             = 1;
+  double          alpha             = 1;
+  double          unit_factor       = 10;
```

Comment:
For consistency, I'd use the same name, and make sure this isn't a magical constant.
```suggestion
  double          length_unit_to_mm = (TGeoManager::GetDefaultUnits() == TGeoManager::kRootUnits) ? 1 / TGeoUnit::mm : 1 / TGeant4Unit::mm;
```

Reply:
And rename `unit_factor` -> `length_unit_to_mm` throughout.

I might even suggest `tgeo_length_unit_in_mm` as a variable name wherever TGeo context is relevant.

Reply:
Yes, including `tgeo` should be more self-explanatory.